### DISCUSSION TITLE
Keymap to hide/unhide cursor, i.e. toggle cursor visibility?

### DISCUSSION THREAD

**Discussion by fooness** [Hammerspoon/hammerspoon#3385] [Q&A]
Hello everyone.

I’m currently trying to migrate from BetterTouchTool to Hammerspoon, and most of my keybindings and window movement functions are already converted.

There’s one function left in my BetterTouchTool configuration which I was not yet able to replicate via Hammerspoon: a keymap to hide/unhide my cursor, i.e. toggle the cursor visibility. Maybe anyone here could help out in these regards.

Thank you!

**Comment by latenitefilms** [comment]
Looks like it'd be pretty simple to add:

https://stackoverflow.com/questions/6350942/how-to-hide-the-mouse-cursor

**Reply by fooness** [reply]
Thank you for your answer. I’m not sure I understand; how would I use this in my hammerspoon config, e.g. within a keymap?
